Example/ios/Podfile

```diff
@@ -15,6 +15,8 @@ target 'ScreensExample' do
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
   use_flipper!
+  pod 'react-native-restart', :path => '../node_modules/react-native-restart'
```

Comment:
Are you sure this and the above in `settings.gradle` is necessary?

Reply:
FIxed in https://github.com/software-mansion/react-native-screens/pull/827/commits/4af99c217e668f87d6ab67bfb8f04773487448c4